pkgs/development/tools/protoc-gen-twirp/default.nix

```diff
@@ -18,6 +18,8 @@ buildGoPackage rec {
     "protoc-gen-twirp_python"
   ];
 
+  doCheck = true;
+
```

Comment:
This is the default for go.

Reply:
For `buildGoPackage` (used here) the `doCheck` default value is [false](https://github.com/NixOS/nixpkgs/blob/c1c617ff8dcb42c05e4e8a20ddc0dde1a50b3dd3/pkgs/development/go-packages/generic/default.nix#L205), for `buildGoModule` its [true](https://github.com/NixOS/nixpkgs/blob/c1c617ff8dcb42c05e4e8a20ddc0dde1a50b3dd3/pkgs/development/go-modules/generic/default.nix#L209) since https://github.com/NixOS/nixpkgs/commit/cc06d7a26f247df071c551a980147baff8a75ff7 / 20.09.

The [latest build](https://hydra.nixos.org/build/141129255) did not run any tests and now [it does](https://logs.nix.ci/?key=nixos/nixpkgs.118924&attempt_id=b5ba4632-f7ba-4a24-aaf1-c80f550b4f09) - at least for the `protoc-gen-twirp` sub-package. The `..._python` sub-package doesn't have any tests.